components/stopwatch.rst

```diff
@@ -48,6 +48,18 @@ You can also provide a category name to an event::
 You can consider categories as a way of tagging events. For example, the
 Symfony Profiler tool uses categories to nicely color-code different events.
 
+Usage in symfony
+----------------
+
+In a Controller, do:
+
+    $stopwatch = $this->get('debug.stopwatch');
+    $stopwatch->start('anEvent');
+
+Or inject `Symfony\Component\Stopwatch\Stopwatch` by autowiring.
```

Comment:
Please use ``:class:`Symfony\\Component\\Stopwatch\\Stopwatch` `` to (a) render it correctly and (b) directly reference it's API documentation.

I think the line below may also be added directly to this paragraph.